### ISSUE TITLE
Integration tests with Dandelion Voting and DissentOracle

### ISSUE THREAD

**Issue by fabriziovigevani** [1Hive/redemptions-app#98]

**Comment by rperez89** [comment]
@fabriziovigevani do you think that is needed to add integration tests? remember that the apps can be installed standalone

**Comment by fabriziovigevani** [comment]
Yeah, but it will be useful to see how the workflow is going to be and will definitely help us identifying new issues that could come up or new ideas as well.
e.g. Implementing token oracle integration tests helped us solidify ACL Oracle permissions knowledge and identify how permissions should be set up.
